src/kOS/Communication/ProcessorManager.cs

```diff
@@ -53,7 +53,7 @@ public void RunProgramOn(List<Opcode> program, Volume volume)
         {
             kOSProcessor processor = GetProcessor(volume);
             var runCommand = new RunCommand {Program = program};
-            processor.ExecuteInterProcCommand(runCommand);            
+            processor.ExecuteInterProcCommand(runCommand);
```

Comment:
I'm thinking we should remove this thing about running a command on another CPU given that it hasn't been updated and doesn't really quite work.  By now so much has changed about the architecture that whatever the old code did would need to be mostly ignored and re-done from scratch anyway.

With the new comms system one could make a "server" program that just receives command-lines as message strings and faithfully logs whatever was in the message out to a filename and then runs the filename.